app/components/cookie_banners_component.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= govuk_cookie_banner(
+      title: 'Cookies on Apply for teacher training',
+      html_attributes: {
+        hidden: false,
+        aria: {
+          label: 'Cookies on Apply for teacher training',
+        },
+        data: {
+          module: 'govuk-fallback-cookie-banner',
+          qa: 'fallback-cookie-banner',
+        },
+      },
+    ) do |cookie_banner| %>
```

Comment:
Probably a very opinionated suggestion, but this indenting looks off to me:

```suggestion
  title: 'Cookies on Apply for teacher training',
  html_attributes: {
    hidden: false,
    aria: {
      label: 'Cookies on Apply for teacher training',
    },
    data: {
      module: 'govuk-fallback-cookie-banner',
      qa: 'fallback-cookie-banner',
    },
  },
) do |cookie_banner| %>
```

Reply:
👍